atef/widgets/config/window.py

```diff
@@ -519,9 +519,6 @@ def __init__(
 
         self.toggle = Toggle()
 
-        # select top level root
-        self.select_by_item(self.root_item.child(0))
```

Comment:
Did this init step never actually need to be done then?

Reply:
It actually gets moved to `DualTree.refresh_model()`, just in a slightly different form.